### PR TITLE
Improve and update French translation

### PR DESCRIPTION
**Pull request by RomFouq**

### CODE REVIEW

File: res/values-fr/strings.xml [hunk 1]
```diff
@@ -2,43 +2,67 @@
...
     <string name="color_is_out_of_moves">%1$s n\'a plus de tours</string>
-    <string name="player_status_moves">%1$d tours</string>
-    <string name="touch_to_skip">Touchez pour omettre</string>
+    <string name="player_status_moves">%1$d tour(s)</string>
+    <string name="touch_to_skip">Toucher pour passer</string>
     <string name="no_player">aucun joueur</string>
-    <string name="not_connected">pas connecté</string>
-    <string name="your_turn">C\'est à vous, %1$s!</string>
+    <string name="not_connected">non connecté</string>
+    <string name="your_turn">À vous de jouer, %1$s&#160;!</string>
```
Comment (shlusiak) [inline res/values-fr/strings.xml]: Are these &#160; necessary? That's the same as &nbsp;, right?

File: res/values-fr/strings.xml [hunk 2]
```diff
@@ -2,43 +2,67 @@
...
+    <string name="do_you_want_to_leave_current_game">Êtes-vous sûr de vouloir quitter la partie en cours&#160;?</string>
+    <string name="disconnect_error">Connexion fermée&#160;: %1$s. Vous êtes maintenant déconnecté.</string>
+    <string name="default_server">Défaut</string>
 
-    <string name="waiting_for_color">Attendant à %1$s</string>
+    <string name="waiting_for_color">En attente de %1$s</string>
     <string name="color_is_out_of_moves">%1$s n\'a plus de tours</string>
-    <string name="player_status_moves">%1$d tours</string>
-    <string name="touch_to_skip">Touchez pour omettre</string>
+    <string name="player_status_moves">%1$d tour(s)</string>
```
Comment (shlusiak) [inline res/values-fr/strings.xml]: Note to self, make this a plural value. ;)

File: res/values-fr/strings.xml [hunk 3]
```diff
@@ -2,43 +2,67 @@
...
+    <string name="googleplus_signout_long">Se connecter avec Google+ en tant que %1$s</string>
+    <string name="googleplus_leaderboard">Meilleurs scores</string>
+    <string name="googleplus_leaderboard_long">Afficher les scores des personnes de vos cercles</string>
+    <string name="googleplus_achievements">Succès</string>
+    <string name="googleplus_achievements_long">Montrer les succès gagnés</string>
 
-    <string name="lobby_waiting_for_players">Attendant aux joueurs</string>
-    <string name="connected_clients">%1$d clients</string>
+    <string name="lobby_waiting_for_players">En attente de joueurs</string>
+    <string name="connected_clients">%1$d client(s)</string>
```
Comment (shlusiak) [inline res/values-fr/strings.xml]: Note to self, make this a plural value.

**Comment by shlusiak** [comment]
Thank you very much, looking great! Are these spaces necessary (# &160) or are you happy for me to trim them afterwards?

**Comment by RomFouq** [comment]
There should be spaces, and they are supposed to be unbreakable ones, but if they cause you technical problems, you can trim them.

**Pull request merged**